index.md

```diff
@@ -98,36 +98,72 @@ Unter Windows kann sich der entsprechende Befehl aus dem `Makefile` entnommen we
 Im Folgenden wird die Installation unter Windows beschrieben.
 Die folgenden Befehle sollten in PowerShell ausgeführt werden (alternativ [pwsh](https://github.com/powershell/powershell)).
 
-Als Package Manager wird Scoop genutzt, da dieser einfache Installationen und Updates ohne administrative Rechte ermöglicht.
+Als Package Manager wird Chocolatey genutzt, da diese einfache Installationen und Updates ermöglichen.
+Eine Shell mit Administrativen rechten wird benötigt! Diese kann mit [Win]+[X], [A] geöffnet werden.
 ```pwsh
 Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
-irm get.scoop.sh | iex
+irm https://get.scoop.sh | iex
+irm https://community.chocolatey.org/install.ps1 | iex
+choco feature enable -n=allowGlobalConfirmation
+```
+`allowGlobalConfirmation` schaltet Rückfragen bei der installation ab.
+
+Zunächst wird die VCS `git` installiert:
+```pwsh
+choco install git
 ```
 
 Weiterhin wird `pip` benötigt um die Python-basierten Extensions zu installieren:
 ```pwsh
-scoop install python
+choco install python.install
 python -m ensurepip
 ```
 
 Nun benötigen wir noch den `make` command aus den GNU coreutils.
-Hier kann beispielsweise die Rust-Implementierung dieser Tools verwendet werden:
+Hierzu werden alle GNU coreutils installiert:
 ```pwsh
-scoop install uutils-coreutils
+choco install gnuwin32-coreutils.install
 ```
```

Comment:
Why does this PR, that is meant to be about VSCode, change the Windows setup to using Chocolatey?

If Chocolatey should be preferred over Scoop is a whole different debate and shouldn't be in this PR.